src/inverseRobot.js

```diff
@@ -7,7 +7,31 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
+    for (let j = 0; j < robotValues.length; j++) {
+      if (i !== j) {
+        if (robotValues[i] === robotValues[j]) {
+          return null;
+        }
+      }
+    }
+  }
+
+  const robotEntries = Object.entries(robot);
+  const objectReversed = {};
+
+  for (let i = 0; i < robotEntries.length; i++) {
+    const innerArray = robotEntries[i].reverse();
+
+    const objectKey = innerArray[0];
+    const objectValue = innerArray[1];
+
+    objectReversed[objectKey] = objectValue;
```

Comment:
Can you do all the checks in this loop?

Reply:
hi, I searched on the internet about how to check if an array contains duplicate elements, and the most easiest and understandable way for me to solve it was javascript Set object, so I came up with that solution.